src/registrar/templates/emails/transition_domain_invitation.txt

```diff
@@ -0,0 +1,11 @@
+You have been invited to manage {% if domains|length > 1 %}multiple domains{% else %}the domain {{ domains.0.name }}{% endif %} on get.gov, 
```

Comment:
@dave-kennedy-ecs I spoke with @Katherine-Osos  today and since @therealslimhsiehdy updated the domain_invitation text in [her newest pr](https://github.com/cisagov/getgov/pull/1026/files#diff-6837eab823a4ed83fd1e89c91d8a372ee3e5b8c8d8c2ba2109c8c8d565af3957), she would like to match the verbiage in there as much as possible. Mind updating this text to follow that PR accordingly?  Feel free though to wait until that PR is merged before updating this.

Reply:
@abroddrick @Katherine-Osos @therealslimhsiehdy I reviewed the pr for 482 yesterday, and I have updated the text.  I added some screenshots to the comment.

Note that this updated text for the email simplifies the code, as I no longer need to add individual urls for each domain to the email.  If you approve these updated screenshots, I will go back and simplify the code on this PR just a bit.